examples/access-control-migration/src/application.ts

```diff
@@ -74,24 +65,11 @@ export class AccessControlApplication extends BootMixin(
   }
 
   setUpBindings(): void {
-    this.bind(TokenServiceBindings.TOKEN_SECRET).to(
-      TokenServiceConstants.TOKEN_SECRET_VALUE,
-    );
-    this.bind(TokenServiceBindings.TOKEN_EXPIRES_IN).to(
-      TokenServiceConstants.TOKEN_EXPIRES_IN_VALUE,
-    );
-    this.bind(TokenServiceBindings.TOKEN_SERVICE).toClass(JWTService);
-
-    this.bind(UserServiceBindings.USER_SERVICE).toClass(MyUserService);
-
     // authorization
     this.bind('casbin.enforcer.factory').to(getCasbinEnforcerByName);
     this.bind('authorizationProviders.casbin-provider')
       .toProvider(CasbinAuthorizationProvider)
       .tag(AuthorizationTags.AUTHORIZER);
-
```

Comment:
There is a possible candidate for the `cabin` component.

Reply:
I will create a follow up PR to upgrade the doc, and refactor it in a separate commit.